src/main/java/cms/gov/madie/measure/resources/MeasureController.java

```diff
@@ -326,6 +326,13 @@ public ResponseEntity<MeasureSet> createCmsId(
         .body(measureSetService.createAndUpdateCmsId(measureSetId, principal.getName()));
   }
 
+  @DeleteMapping("/measures/{measureId}/delete-cms-id")
+  public ResponseEntity<String> deleteCmsId(
+      @PathVariable String measureId, @RequestParam(name = "cmsId") Integer cmsId) {
```

Comment:
Since this is a request that will be coming from Helpdesk and we have to manually trigger the end point, Do you think we could add any api-key to validate the request and add a level of security ?

Reply:
good catch! it needs to be protected with api-key

Reply:
Added! Thank you